cypress/e2e/awx/dashboard/welcomeModal.cy.tsx

```diff
@@ -4,14 +4,14 @@ describe('dashboard: Welcome modal', () => {
   });
 
   it('renders the Welcome Modal on the dashboard', () => {
-    cy.visit(`/ui_next/dashboard`);
+    cy.navigateTo(/^Dashboard$/);
     cy.getDialog().within(() => {
       cy.contains('Welcome to the new Ansible user interface').should('be.visible');
     });
   });
 
   it('renders the Welcome Modal on reload of the dashboard until checkbox is selected', () => {
-    cy.visit(`/ui_next/dashboard`);
+    cy.navigateTo(/^Dashboard$/);
```

Comment:
Please revert.

Reply:
We should not be using hard coded urls in tests.